doc/dev/design/ui_goals.rst

```diff
@@ -0,0 +1,57 @@
+Ceph Dashboard Design Goals
+===========================
+
+.. note:: this document is intended to provide a focal point for discussing the overall design principles for mgr/dashboard
+
+Introduction
+############
+
+Most distributed storage architectures are inherently complex, and can present a management challenge to Operations teams who are
+typically stretched across multiple product and platform disciplines. In general terms, the complexity of any solution can have a 
+direct bearing on the operational costs incurred to manage it. The answer is simple...make it simple :)
+
+This document is intended to highlight Ceph Dashboard design goals which may help to  
+
+* reduce complexity
+* increase productivity
+* improve time-to-value
+* increase observability
+
+ 
+Understanding the Persona of the Target User
+############################################
+
+Ceph has historically been administered from the CLI. The CLI has always and will always offer the richest, most flexible way to 
+install and manage a Ceph cluster. Administrators who require and demand this level of control are unlikely to adopt a UI for anything
+more than a technical curiousity.
+
+The relevance of the UI is therefore more critical for a new SysAdmin, where it can help technology adoption and reduce the operational friction
+that is normally experienced when implementing a new solution.
+
+Understanding the target user persona is therefore a fundamental first step in design. Attempting to design a UI that meets the requirements of a 'seasoned'
+Ceph Administrator or Developer, and a relatively new SysAdmin is unlikely to satisfy either user group.
+
+Design Principles
+#################
+
+Key Principles
+--------------
+
+
+#. **Clarity and consistency**. The UI should ensure the data shown is unambiguous and consistent across different views
+#. **Data timeliness**. Data displayed in the UI must be timely. State information must always be reasonably recent for it to be relevant and acted upon with confidence.
+#. **Automate through workflows**. If the admin has to follow a 'recipe' to perform a task, the goal of the dashboard UI should be to implement the flow.
+#. **Provide a natural next step**. The UI **is** the *expert system*, so instead of expecting the user to know where to they go next, the UI should lead them. This means linking
+   components together to establish a flow, and deeper integration between the alertmanager implementation and the dashboard elements enabling an Admin to efficiently step from alert to 
+   affected component.
+#. **Platform visibility**. The platform is a fundamental component of the Ceph solution, so providing some insights into the underlying platform can help maintain user focus with the UI.
```

Comment:
Not sure what is mean by "the platform" here - would you mind elaborating on this?

Reply:
ACK - platform = OS + hardware. changed.